tests/acceptance/features/apiSharingNg/shareInvitations.feature

```diff
@@ -924,3 +924,59 @@ Feature: Send a sharing invitations
       | Co Owner         | folder        | FolderToShare  |
       | Uploader         | folder        | FolderToShare  |
       | Manager          | folder        | FolderToShare  |
+
+
+  Scenario Outline: send sharing invitation to unknown group
+    Given user "Alice" has uploaded file with content "to share" to "/textfile1.txt"
+    And user "Alice" has created folder "FolderToShare"
+    When user "Alice" sends the following share invitation using the Graph API:
+      | resourceType    | <resource-type>    |
+      | resource        | <path>             |
+      | space           | Personal           |
+      | sharee          | unknownGroup       |
+      | shareType       | group              |
+      | permissionsRole | <permissions-role> |
+    Then the HTTP status code should be "400"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "error"
+        ],
+        "properties": {
+          "error": {
+            "type": "object",
+            "required": [
+              "code",
+              "message"
+            ],
+            "properties": {
+              "code": {
+                "type": "string",
+                "enum": [
+                  "generalException"
+                ]
+              },
+              "message": {
+                "type": "string",
+                "enum": [
+                  "itemNotFound: not found"
+                ]
+              }
+            }
+          }
+        }
+      }
+      """
+    Examples:
+      | permissions-role | resource-type | path           |
+      | Viewer           | file          | /textfile1.txt |
+      | File Editor      | file          | /textfile1.txt |
+      | Co Owner         | file          | /textfile1.txt |
+      | Manager          | file          | /textfile1.txt |
+      | Viewer           | folder        | FolderToShare  |
+      | Editor           | folder        | FolderToShare  |
+      | Co Owner         | folder        | FolderToShare  |
+      | Uploader         | folder        | FolderToShare  |
+      | Manager          | folder        | FolderToShare  |
```

Comment:
all role should not be allowed to file resource see issue https://github.com/owncloud/ocis/issues/8131

Reply:
Currently every role can be assigned on the file, and this same example table is being used in every other Scenario. What should we do about it? Should we do it as it is or should we wait for some fixes?
cc @amrita-shrestha @saw-jan

Reply:
I really don't know what roles should be permitted while sharing a file/folder. If it is known then we can make another PR for removing not permitted roles and add some coverage for trying to share with not permitted roles. Then block that PR until the fix arrives
But I don't think  we need to block this one.